UKHO.PeriodicOutputService/UKHO.PeriodicOutputService.API.FunctionalTests/FunctionalTests/POSAIOScenarioWithInvalidProductIdentifier.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System.Net;
+using FluentAssertions;
+using NUnit.Framework;
+using UKHO.PeriodicOutputService.API.FunctionalTests.Helpers;
+using static UKHO.PeriodicOutputService.API.FunctionalTests.Helpers.TestConfiguration;
+
+namespace UKHO.PeriodicOutputService.API.FunctionalTests.FunctionalTests
+{
+    [Category("POSAIOScenarioWithInvalidProductIdentifier")]
+    public class POSAIOScenarioWithInvalidProductIdentifier
+    {
+        private string fssJwtToken;
+        private static readonly POSWebJobApiConfiguration posWebJob = new TestConfiguration().POSWebJobConfig;
+        private static readonly POSFileDetails posDetails = new TestConfiguration().posFileDetails;
```

Comment:
these seem to be repeated, we need to pull them out and have a common area so that it makes support and future maintenance easier and cost effective